packages/sources/finage/src/endpoint/etf.ts

```diff
@@ -25,24 +29,30 @@ export interface ResponseSchema {
 }
 
 export const makeEtfExecute =
-  (executeConfig: { country?: string }): ExecuteWithConfig<Config> =>
+  (country?: string): ExecuteWithConfig<Config> =>
   async (request, _, config) => {
     const validator = new Validator(request, inputParameters, {}, { overrides })
 
     const jobRunID = validator.validated.id
     const base = validator.validated.data.base
     const symbol = validator.overrideSymbol(NAME, base).toUpperCase()
+    const param_country = validator.validated.data.country
+      ? validator.validated.data.country.toLocaleLowerCase()
+      : country
+      ? country
+      : null
```

Comment:
Minor refactor - I left out the `null` based on a later comment, but it could be added if needed.

```suggestion
    const param_country = validator.validated.data.country?.toLocaleLowerCase() || country
```